app/controllers/application_controller.rb

```diff
@@ -26,7 +26,10 @@ def patron
     MAX_INACTIVE_TIME = 2.hours - 5.minutes
 
     def authenticate_webaccess
-      redirect_to Settings.symws.webaccess_url + request.base_url
+      # if we aren't given a REMOTE_USER variable we are unauthorized.
+      # this maybe should be a 401, however if apache is misconfigured
+      # there's no amount of retrying that will fix this for the guest
+      redirect_to '/500' unless request.env.fetch(Settings.remote_user_header, nil)
```

Comment:
we can redirect to a custom error page for this case as you mentioned @whereismyjetpack, we can add that in a later PR 👍  I think it is good for now, we all would love to see this PR deployed.